lg_common/scripts/actual_button.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+import tkinter as tk
+from tkinter import ttk
+import threading
+import signal
+import sys
+import rospy
+from std_msgs.msg import String
+
+
+class ButtonApp(threading.Thread):
+    def onclick(*args, **kwargs):
+        p = rospy.Publisher('/touchscreen/toggle', String, queue_size=10)
```

Comment:
The 100ms sleep here is unnecessary delay for a few frames.   If you keep a Publisher instance around you won't need to wait for connections, and it won't hide itself from ROS topic tools.